src/pages/messages/components/NviCandidateActions.tsx

```diff
@@ -199,15 +201,17 @@ export const NviCandidateActions = ({ nviCandidate, nviCandidateQueryKey }: NviC
 
         {myApproval?.status !== 'Approved' && (
           <>
-            <Typography gutterBottom>{t('tasks.nvi.approve_nvi_candidate_description')}</Typography>
+            <Typography fontWeight="bold">{t('tasks.nvi.nvi_status')}</Typography>
+            <Trans i18nKey="tasks.nvi.approve_nvi_candidate_description" components={[<Typography paragraph />]} />
```

Comment:
For å få lik spacing over godkjenn- og avvis-knappene. Kan også vurdere å erstatte gutterBottom med paragraph på denne og linje 223 ('tasks.nvi.reject_nvi_candidate_description'), da det gir litt mer pusterom (noe jeg synes er bedre, men det er subjektivt).

```suggestion
            <Trans i18nKey="tasks.nvi.approve_nvi_candidate_description" components={[<Typography gutterBottom />]} />
```

Reply:
yeah, enig i at mer pusterom er bedre